.gitignore

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```

Comment:
You have error during merge conflict. Please pay attention and fix it

Reply:
+